### ISSUE TITLE
feat: use `--bundle` mode by default

### ISSUE THREAD

**Issue by ctjlewis** [tsmodule/tsmodule#32]
This lets "devDeps by default" out of the box make more sense; packages ship with fewer dependencies, more frequently as inlined bundles.

**Comment by ctjlewis** [comment]
Blocked by #33.

**Comment by ctjlewis** [comment]
Probably won't do this for cosmetic reasons. Easy to add `-b` if you definitely want a bundle.
